live-build/misc/migration-scripts/dx_apply

```diff
@@ -64,7 +64,7 @@ ARCHIVE_DIR="$1"
 	die "$ARCHIVE_DIR does not have a version.info file"
 
 . "$ARCHIVE_DIR/version.info"
-[[ "$DLPX_MIN_VERSION" == "5.3.5.0" ]] ||
+[[ "$DLPX_MIN_VERSION" == "5.3.6.0" ]] ||
 	die "expected DLPX_MIN_VERSION for migration to be 5.3.5.0"
```

Comment:
Message needs to be updated. I'm wondering if it doesn't make more sense to make these strings as variables at the top of the file so that we don't have to update all the messages, etc.